docs/walkthrough.mdx

```diff
@@ -229,7 +229,7 @@ Suppose you wish to deploy a network comprising three nodes, where two nodes act
 Generate the required configuration for these nodes by running:
 
 ```bash
-kwil-admin setup testnet -v 2 -n 1 --hostname "192.168.2.10" --hostname "192.168.2.20" --hostname "192.168.2.10" -o "./testnet"
+kwil-admin setup testnet -v 2 -n 1 --hostnames "192.168.2.10" "192.168.2.20" "192.168.2.30" -o "./testnet"
 ```
 
 This generates `private_key`, `genesis.json` and `config.toml` files for all the 3 nodes at location `./testnet`.
```

Comment:
Looks right now.  We could however clarify here like "...for all the 3 nodes (2 validators and 1 non-validator) at location..."  because it's probably not clear what those numbers mean.